src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -249,22 +228,26 @@ private void readRcaEnabledFromConf() {
    * has changed in the meantime (such as a new elected master). It also starts the RCA runtime if
    * it wasn't already running but the current state of the flag expects it to.
    */
-  private void startRcaNanny() {
+  public void startNanny() {
+    startNodeRolePoller();
```

Comment:
I tend to think, it we should move it inside the `startPollers()`, so now just two instead of three